sql/99-patches.sql

```diff
@@ -294,5 +294,74 @@ $P$
 );
 
 
+--------------------------------------------------------------------------------
+-- protect_reference column VARCHAR(100) > VARCHAR(255)
+--
+-- For tables bde.crs_title & bde.crs_ttl_inst_protect
+-- as per https://github.com/linz/linz-bde-uploader/issues/279
+--------------------------------------------------------------------------------
+
+PERFORM _patches.apply_patch(
+    'protect_reference column VARCHAR(100) > VARCHAR(255)',
+    $P$
+DO $$
+DECLARE
+  v_isversioned BOOL;
+BEGIN
+
+  v_isversioned := false;
+
+  IF EXISTS (SELECT p.oid
+             FROM pg_catalog.pg_proc p, pg_catalog.pg_namespace n
+             WHERE p.proname = 'ver_is_table_versioned'
+             AND n.oid = p.pronamespace
+             AND n.nspname = 'table_version')
+  THEN
+    v_isversioned := true;
+  END IF;
+
+  -- Alter bde.crs_ttl_inst_protect
+  ALTER TABLE bde.crs_ttl_inst_protect
```

Comment:
ver_versioned_table_change_column_type will change the main table column data type for you. So this should only be run if that table is not versioned.